.changeset/nasty-horses-enjoy.md

```diff
@@ -0,0 +1,15 @@
+---
+"@promster/metrics": patch
+"@promster/server": patch
+"@promster/types": patch
+---
+
+# Breaking Changes
```

Comment:
It should be `major` right?
I think changeset bumps a major anyway when peer deps change

Reply:
Thanks.